tests/browserify-public/browserify-bundle.js

```diff
@@ -1311,17 +1318,6 @@ Interceptor.prototype.replyWithFile = function replyWithFile(statusCode, filePat
     return this.reply(statusCode, readStream, headers);
 };
 
-Interceptor.prototype.replyWithFile = function replyWithFile(statusCode, filePath, headers) {
-    if (! fs) {
-        throw new Error('No fs');
-    }
-    var readStream = fs.createReadStream(filePath);
-    readStream.pause();
-    this.filePath = filePath;
-    return this.reply(statusCode, readStream, headers);
-};
-
-
```

Comment:
This file is updated every time the tests are run. I'm trying to figure out a good enough way to avoid it muddling with the diffs (probably just gitignore it for a start).